### PR TITLE
nef - module markdown

### PR DESCRIPTION
**Pull request by miguelangel-dev**
## Description
We want to open `nef` to developers. On this way, `nef` should be easily imported into Xcode projects to expand its power. In this PR we have open markdown API.

## How does it work?
You can import the project using SPM with the next URL: https://github.com/bow-swift/nef.git
After you can `import nef` and use the open API 

Captura de pantalla 2019-07-10 a las 13 50 30

### CODE REVIEW

File: project/Component/NefCarbon/App/Assets.swift [hunk 1]
```diff
@@ -7,7 +7,7 @@ enum Assets {
...
...NS4yICg3ODE4MSkgLSBodHRwczovL3NrZXRjaGFwcC5jb20gLS0+CiAgICA8dGl0bGU+NTEyeDUxMjwvdGl0bGU+CiAgICA8ZGVzYz5DcmVhdGVkIHdpdGggU2tldGNoLjwvZGVzYz4KICAgIDxnIGlkPSI1MTJ4NTEyIiBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4KICAgICAgICA8ZyBpZD0iR3JvdXAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDc0LjAwMDAwMCwgMzEuMDAwMDAwKSI+CiAgICAgICAgICAgIDxwb2x5Z29uIGlkPSJwYXRoIiBzdHJva2U9IiNGRkZGRkYiIHN0cm9rZS13aWR0aD0iNy41NDQ0Nzg2MSIgcG9pbnRzPSIwIDM0NC41NjM5MzEgMTgxLjk5Njc1IDQ1MCAzNjQgMzQ0LjU1NzQwOCAxODEuOTk2NzUgMjM5LjEyMTMzOSI+PC9wb2x5Z29uPgogICAgICAgICAgICA8cG9seWdvbiBpZD0icGF0aCIgc3Ryb2tlPSIjRkZGRkZGIiBzdHJva2Utd2lkdGg9IjcuNTQ0NDc4NjEiIHBvaW50cz0iMCAzMTQuNDMxODg1IDE4MS45OTY3NSA0MTkuODc0NDc3IDM2NCAzMTQuNDMxODg1IDE4MS45OTY3NSAyMDguOTk1ODE2Ij48L3BvbHlnb24+CiAgICAgICAgICAgIDxwb2x5Z29uIGlkPSJwYXRoIiBzdHJva2U9IiNGRkZGRkYiIHN0cm9rZS13aWR0aD0iNy41NDQ0Nzg2MSIgcG9pbnRzPSIxODEuOTk2NzUgMCAwIDI4NC44MTYyOTQgMTgxLjk5Njc1IDM4OS43NDg5NTQgMzY0IDI4NC44MTYyOTQiPjwvcG9seWdvbj4KICAgICAgICAgICAgPHBvbHlnb24gaWQ9InBhdGgtY29weSIgZmlsbC1vcGFjaXR5PSIwLjIiIGZpbGw9IiNGRkZGRkYiIHBvaW50cz0iMTgxLjA1Njk5NSAwIDE4MS4wNTY5OTUgMzg5Ljc0ODk1NCAzNjQgMjg0LjgxNjI5NCI+PC9wb2x5Z29uPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+
```
Comment (truizlop) [inline project/Component/NefCarbon/App/Assets.swift:10]: Very readable!

File: project/Component/nef/nef.swift [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
 
 /// Renders a code selection into multiple Carbon images.
 ///
 /// - Precondition: this method must be invoked from main thread.
 /// - Postcondition: you should manage the output `NSWindow`.
 ///
 /// - Parameters:
-///   - code: content to generate the snippet.
-///   - style: style to apply to exported code snippet.
+///   - code: content for generation the snippet.
```
Comment (truizlop) [inline project/Component/nef/nef.swift]:
```suggestion
///   - code: content to generate the snippet.
```

File: project/Component/nef/nef.swift [hunk 3]
```diff
@@ -44,12 +68,12 @@ public func carbon(code: String,
 /// - Precondition: this method must be invoked from main thread.
 ///
 /// - Parameters:
-///   - parentView: canvas view where to render Carbon image.
-///   - code: content to generate the snippet.
-///   - style: style to apply to exported code snippet.
+///   - parentView: canvas works where to render Carbon image.
+///   - code: content for generation the snippet.
```
Comment (truizlop) [inline project/Component/nef/nef.swift]:
```suggestion
///   - code: content to generate the snippet.
```

**Review by truizlop — APPROVED** [review]

File: project/Component/NefCarbon/App/Assets.swift (hunk 1 above)
Comment (miguelangel-dev) [reply project/Component/NefCarbon/App/Assets.swift:10]: can not you see the icon?

**Pull request merged**